### PR TITLE
Fix QuadGrid class

### PR DESCRIPTION
**Pull request by Jesus89**
Check geometry in the input_gdf and keep the original geometry column name.

### CODE REVIEW

File: cartoframes/analysis/grid.py [hunk 1]
```diff
@@ -1,27 +1,33 @@
...
 
-        dfs = []
+        geometry_name = input_gdf.geometry.name
 
-        for _, row in input_gdf.iterrows():
-            bounds = row['geometry'].bounds
+        dfs = []
+        for index, row in input_gdf.iterrows():
+            input_geometry = input_gdf.geometry[index]
+            bounds = input_geometry.bounds
```
Comment (alasarr) [inline cartoframes/analysis/grid.py]: Instead of doing this how about: 

```
bounds = row[geometry_name].bounds
```
```

Comment (Jesus89) [reply cartoframes/analysis/grid.py]: I have just implemented this change :)

We must keep `input_geometry` because it is used later

**Comment by Jesus89** [comment]
Already done in https://github.com/CartoDB/cartoframes/pull/1338/commits/4c68e46ff69dbd74f7551e58f439afe7d2d3af10

**Comment by alasarr** [comment]
Acceptance done and working at Collab!

**Pull request merged**